manifests/repo/el.pp

```diff
@@ -26,4 +28,11 @@
       enabled  => 1,
     }
   }
+
+  if $yum_proxy {
+    Yumrepo['jenkins'] {
+      proxy => $yum_proxy,
+    }
```

Comment:
not necessarily. this was the first solution that came to my mind without thinking to much about it...

i could move the proxy parameter directly to the yumrepo resource declarations. should we keep 'undef' for the default case or should we convert the parameter to 'absent' in el.pp?

i was also thinking about the name 'yum_proxy'. maybe we should call it 'repo_proxy' as this could also be used for apt and zypper repos. 

thanks for you feedback